pkgs/applications/office/gnucash/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p curl jq nix-prefetch-github
+
+set -euo pipefail
+
+latest_version=$(curl -s https://api.github.com/repos/Gnucash/gnucash/releases/latest | jq -r '.tag_name')
+
+if [[ "$latest_version" = "$UPDATE_NIX_OLD_VERSION" ]]; then
+    echo "already up to date"
+    exit 0
+fi
+
+old_src_hash=$(nix-instantiate --eval -A gnucash.src.outputHash | tr -d '"')
+old_src_doc_hash=$(nix-instantiate --eval -A gnucash.docs.src.outputHash | tr -d '"')
+
+src_hash=$(nix-prefetch-url https://github.com/Gnucash/gnucash/releases/download/$latest_version/gnucash-$latest_version.tar.bz2)
+src_hash=$(nix-hash --to-sri --type sha256 "$src_hash")
+src_doc_hash=$(nix-prefetch-github Gnucash gnucash-docs --rev "$latest_version" | jq -r .sha256)
```

Comment:
When I run `nix-prefetch-github`, I don't get a `sha256` field.  I get a `hash` field instead.  I'm not sure why it's different.  This causes the update to fail:

		$ nix-shell maintainers/scripts/update.nix --argstr package gnucash
		...
		Running update for:
		 - gnucash-5.4: UPDATING ...
		 - gnucash-5.4: ERROR

		--- SHOWING ERROR LOG FOR gnucash-5.4 ----------------------
		...
		+ src_hash=sha256-tNr2e7iStwYyP2Lp+pckIDnX3QouHhB3HgwlgX3Q7Ts=
		++ nix-prefetch-github Gnucash gnucash-docs --rev 5.5
		++ jq -r .sha256
		+ src_doc_hash=null
		++ nix-hash --to-sri --type sha256 null
		error: hash 'null' has wrong length for hash type 'sha256'
		+ src_doc_hash=

You can make it work either way by using `jq -r '.hash // .sha256'`

Reply:
Oh, nix-prefetch-github [changed](https://github.com/seppeljordan/nix-prefetch-github#v700) its output format in 7.0.0, I'm seeing the same issue after updating channels.

I set `-I nixpkgs=./.` and left just `.hash` for now, please tell me if you think leaving both would be better.

Reply:
Well, 23.05 support [just ended 2023-12-31](https://nixos.org/manual/nixos/stable/release-notes#sec-release-23.05), so requiring that package maintainers running update scripts be on a supported release (or unstable/master) seems fine.